### PR TITLE
Added missing properties for databases and server

### PR DESCRIPTION
**Pull request by anthonydresser**
Added Last backup date and last log backup date, and machine name

### CODE REVIEW

File: src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs [hunk 1]
```diff
@@ -730,6 +744,8 @@ public bool HasSameValueAs(DatabaseData other)
             {
                 bool result =
                     (this.recoveryModel == other.recoveryModel) &&
+                    (DateTime.Compare(this.lastBackupDate, other.lastBackupDate) == 0) &&
```
Comment (benrr101) [inline src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs]: It isn't necessary to use the `DateTime.Compare`, you should be able to just use `==`

File: src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs [hunk 2]
```diff
@@ -730,6 +744,8 @@ public bool HasSameValueAs(DatabaseData other)
...
+                    (DateTime.Compare(this.lastLogBackupDate, other.lastLogBackupDate) == 0) &&
```
Comment (benrr101) [inline src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs]: Also, all these parens are extraneous, but I can see the desire to stick with the existing pattern

File: src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs [hunk 3]
```diff
@@ -892,6 +908,40 @@ public virtual RecoveryModel RecoveryModel
...
+        public DateTime LastBackupDate
+        {
+            get
+            {
+                return this.currentState.lastBackupDate;
+            }
+            set
+            {
+                this.currentState.lastBackupDate = value;
+                this.NotifyObservers();
```
Comment (benrr101) [inline src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs]: I think ideally if this is notifying observers when a property is being updated, it should be implementing the INotifyPropertyChanged interface. But, I also assume this is a huge change, so I guess we can leave it be.

Comment (anthonydresser) [inline src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs]: Yeah I was mostly just doing what other properties were doing.

File: src/Microsoft.SqlTools.ServiceLayer/Admin/AdminServicesProviderOptionsHelper.cs [hunk 4]
```diff
@@ -32,6 +32,9 @@ public class AdminServicesProviderOptionsHelper
         internal const string DatabaseContainmentType = "databaseContainmentType";
         internal const string DatabaseState = "databaseState";
         internal const string RecoveryModel = "recoveryModel";
+        internal const string CompatabilityLevel = "compatabilityLevel";
```
Comment (kevcunnane) [inline src/Microsoft.SqlTools.ServiceLayer/Admin/AdminServicesProviderOptionsHelper.cs]: compatibility

**Pull request merged**